official/resnet/ctl/ctl_imagenet_main.py

```diff
@@ -296,6 +316,5 @@ def main(_):
   logging.set_verbosity(logging.INFO)
   common.define_keras_flags()
   ctl_common.define_ctl_flags()
-  flags.adopt_module_key_flags(keras_common)
```

Comment:
Why remove this line?

Reply:
The `keras_common` is not defined in this file. We will encounter `undefined error` if it is not removed.